src/components/InventoryV2/InventoryListByBatch.tsx

```diff
@@ -132,14 +127,15 @@ export default function InventoryListByBatch({ setReportData }: InventoryListByB
   const location = useStateLocation();
   const updateFilterQueryParams = useCallback(
     (f: InventoryFiltersType) => {
-      query.set('facilityId', f.facilityIds?.map((n) => n.toString())?.join(',') ?? '');
-      query.set(
-        'subLocationName',
-        f.subLocationsIds?.map((id) => subLocations?.find((sl) => sl.id === id)?.name ?? '')?.join(',') ?? ''
+      const newQuery = new URLSearchParams();
+      f.facilityIds?.forEach((n) => newQuery.append('facilityId', n.toString()));
+      f.subLocationsIds?.forEach((id) =>
+        newQuery.append('subLocationName', subLocations?.find((sl) => sl.id === id)?.name ?? '')
```

Comment:
optionally sublocationIds->map to name->filter on non-empty name->foreach append to newQuery

Reply:
Okay, yes, makes sense to filter empty names here